app/src/main/java/org/simple/clinic/patient/OngoingNewPatientEntry.kt

```diff
@@ -118,7 +119,7 @@ data class OngoingNewPatientEntry(
       copy(
           personalDetails = PersonalDetails(
               fullName = patientProfileInfo.fullName,
-              dateOfBirth = null,
+              dateOfBirth = patientProfileInfo.dateOfBirth.format(DateTimeFormatter.ofPattern("dd/MM/yyyy")),
```

Comment:
Please define date formatters in `DateFormatterModule` and pass them to the method as a parameter instead of hardcoding them here.

In fact, we already have a formatter for this specific format already defined.